exercises/react/test/test_react.c

```diff
@@ -0,0 +1,295 @@
+#include <stddef.h>
+#include <stdio.h>
+#include "vendor/unity.h"
+#include "../src/react.h"
+
+void test_input_cells_have_value(void)
+{
+   struct reactor *r = create_reactor();
+   struct cell *input = create_input_cell(r, 2);
+
+   TEST_ASSERT_EQUAL_INT(2, cell_value(input));
+
+   free_reactor(r);
+}
+
+static int plus1(int x)
+{
+   return x + 1;
+}
+
+void test_compute_cells_calculate_initial_value(void)
+{
+   struct reactor *r = create_reactor();
+   struct cell *input = create_input_cell(r, 1);
+   struct cell *output = create_compute1_cell(r, input, plus1);
+
+   TEST_ASSERT_EQUAL_INT(2, cell_value(output));
+
+   free_reactor(r);
+}
+
+static int concat_digits(int a, int b)
```

Comment:
lot of functions scattered throughout the test file. I wonder if they should all be defined at the top, or rather defined close to where they are used.

I defined them as close to where they are used as possible.